content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md

```diff
@@ -466,7 +483,7 @@ Setting the ingress IP depends on the cluster provider:
 1.  _Other environments:_
 
     {{< text bash >}}
-    $ export INGRESS_HOST=$(kubectl get po -l istio=ingressgateway -n istio-system -o jsonpath='{.items[0].status.hostIP}')
+    $ export INGRESS_HOST=$(kubectl get po -l "${INGRESS_NAME}" -n  "${INGRESS_NS}" -o jsonpath='{.items[0].status.hostIP}')
```

Comment:
Does this need an `istio=` or maybe it was wrong originally.

Reply:
Thanks, good catch! My change here is wrong.